Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.0
```

Comment:
Why is the min req swift version lowered on this PR?

Reply:
Because the CI is building with Swift 5.0. The entire build config will have to be updated in a future PR to match the other iOS repos, but for now these changes will do.